bootstrap/15-locale.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright (C) 2017 Denis Mosolov <denismosolov@cusdeb.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
```

Comment:
In this script you rely on the `TIME_ZONE` variable which for some reason can be undefined. Use

```bash
if ! check_if_variable_is_set TIME_ZONE; then
    >&2 echo "TIME_ZONE is not specified"
    exit 1
fi
```

to protect the script from unpredictable behaviour if the variable is unset.

Reply:
I've missed TIME_ZONE=${TIME_ZONE:='Etc/UTC'}

Reply:
It assigns a default value here:  https://github.com/tolstoyevsky/pieman/blob/timezone/pieman.sh#L76
I cannot imagine a case when `TIME_ZONE` can be undefined.